kvbc/test/sparse_merkle_storage/kv_blockchain_db_editor_test.cpp

```diff
@@ -435,6 +435,64 @@ TEST_F(DbEditorTests, get_block_key_values) {
   ASSERT_THAT(out_.str(), EndsWith("\n}\n"));
 }
 
+TEST_F(DbEditorTests, get_latest_category_updates_merkle) {
+  ASSERT_EQ(EXIT_SUCCESS,
+            run(
+                CommandLineArguments{
+                    {kTestName, rocksDbPath(main_path_db_id_), "getEarliestCategoryUpdates", kCategoryMerkle}},
```

Comment:
I think the test name is incorrect, because the command we call is `getEarliestCategoryUpdates()` and the test is called `latest`.

Reply:
fixed